screwdriver-cd/launcher
# fix: Pull in latest store-cli and meta-cli

## Context

This PR pulls in latest store-cli: 
- Restore timestamps when restoring caches https://github.com/screwdriver-cd/store-cli/pull/43
- Enable versioning https://github.com/screwdriver-cd/store-cli/pull/44
And latest meta-cli (https://github.com/screwdriver-cd/meta-cli/compare/v0.0.22...v0.0.26):
- Tests
- Fix staticcheck issues in meta.go
- Ensure global options and commands are not duplicated

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.
